custom_components/test/fuzzing/generators/test_gen_on_xxx.py

```diff
@@ -0,0 +1,44 @@
+import pytest
+import logging
+
+from custom_components.test.fuzzing.fuzzer_utils.ValuePoolFuzzer import ValuePoolFuzzer
+from custom_components.test.fuzzing.fuzzer_utils.GeneratorFuzzer import GeneratorFuzzer
+from custom_components.test.fuzzing.fuzzer_utils.GeneratorRunner import GeneratorRunner
+from custom_components.test.fuzzing.generators.DummyClass import DummyClass
+
+
+# Logger setup
+logging.basicConfig(level=logging.INFO)
```

Comment:
- The Logger config is specified globally in the `.pytest.ini` file

Reply:
Thanks, I have removed the line